tests/unit/executors/test_local_executor.py

```diff
@@ -8,7 +8,7 @@
 class TestLocalExecutor:
     @pytest.fixture
     def executor(self):
-        return LocalExecutor()
+        return LocalExecutor(install_dependencies_on_execute=True)
```

Comment:
@vasinov not sure what the best way to test this new functionality is. Any ideas?